terraform/app/modules/paas/inputs.tf

```diff
@@ -54,7 +54,7 @@ variable "web_app_memory" {
 variable "web_app_disk_quota" {
   type        = number
   description = "Application disk quota"
-  default     = 2048
+  default     = 4096
```

Comment:
What was the reason for this again?

Reply:
There are three branches for ER-738. Could you point me at the workflow run that failed please

Reply:
I think we should dig a little deeper before resorting to doubling the disk quota.

Reply:
I disagree, but we should create a ticket so the investigation can be prioritized.

Reply:
[This PR](https://github.com/DFE-Digital/early-years-foundation-recovery/pull/789) has [deployed](https://ey-recovery-pr-789.london.cloudapps.digital/) successfully and you can cherry pick [this commit](https://github.com/DFE-Digital/early-years-foundation-recovery/pull/789/commits/bbd76c6c114307d41a22d6fe29a6c7bdbc85aa44) to mitigate the issue.

## Summary

Disk size bloat prevented deployment because our quota was set to 2GB.

```
$ docker images | grep recovery

main> 
prod    1.64GB

ER738> 
prod    2.97GB
```

Comparing disk usage between these changes and `main` identifies the culprit

```
$ docker run -it --entrypoint sh recovery:prod 

main> du -h -d 1 /
        1.1M    /bin
        1.3M    /etc
        4.0K    /mnt
        16.0K   /media
        2.1M    /tmp
        4.0K    /opt
        6.7M    /lib
        0       /dev
        233.2M  /srv    <-----
        108.0K  /sbin
        24.0K   /root   <-----
        0       /proc
        1.4G    /usr
        4.0K    /run
        0       /sys
        4.0K    /home
        120.0K  /var
        4.0K    /lib64
        1.6G    /


ER738> du -h -d 1 /
    1.1M    /bin
    1.3M    /etc
    4.0K    /mnt
    16.0K   /media
    2.1M    /tmp
    4.0K    /opt
    6.7M    /lib
    0       /dev
    579.9M  /srv    <------
    108.0K  /sbin
    1.0G    /root   <------
    0       /proc
    1.4G    /usr
    4.0K    /run
    0       /sys
    4.0K    /home
    124.0K  /var
    4.0K    /lib64
    3.0G    /


```

[dfe-frontend-alpha](https://www.npmjs.com/package/dfe-frontend-alpha) has dev dependencies in the runtime group pulling in additional code that is not used in deployments. When this is eventually resolved we can expect `node_modules` to be leaner but as it stands the image is `1.88GB`.

https://github.com/microsoft/playwright

> Playwright is a framework for Web Testing and Automation. It allows testing Chromium, Firefox and WebKit with a single API. 

https://github.com/garris/BackstopJS

> BackstopJS automates visual regression testing of your responsive web UI by comparing DOM screenshots over time.

```
/srv # du -h -d 1 /root/.cache/ms-playwright/

342.9M  /root/.cache/ms-playwright/webkit-1869
5.5M    /root/.cache/ms-playwright/ffmpeg-1009
8.0K    /root/.cache/ms-playwright/.links
223.0M  /root/.cache/ms-playwright/firefox-1419
472.5M  /root/.cache/ms-playwright/chromium-1071
1.0G    /root/.cache/ms-playwright/
```